src/components/CommentList.jsx

```diff
@@ -3,7 +3,10 @@ import axios from 'axios'
 
 export const CommentList = ({ id, article }) => {
 	const [comments, setComments] = useState([])
-	const date = article.created_at.replace('T', ' ').replace('Z', '')
+	const date = article.created_at
+		.replace('T', ' ')
+		.replace('Z', '')
+		.slice(0, 16)
 
```

Comment:
if you're going to do this to all the dates then this can definitely be extracted away into a utility function